session/server.go

```diff
@@ -62,6 +63,7 @@ func (m *mailboxSession) run(mailboxServer *mailbox.Server) {
 
 func (m *mailboxSession) stop() {
 	m.server.Stop()
+	close(m.quit)
```

Comment:
Is this closed in the simple case of a mailbox server restart? Haven't seen an instance where we'll re-establish our connections when that flaps...